util.py

```diff
@@ -373,6 +374,9 @@ def setLocalTime(timestring, timezone=None):
 
     assert runCmd2("date --set='%s'" % timestring) == 0
 
+def getMajorVersion():
```

Comment:
Since master is XS9 then the usages of this will disappear so this can be removed too